docs/source/contributing/acceptance-tests.md

```diff
@@ -7,80 +7,115 @@ myst:
     "keywords": "Volto, Plone, frontend, React, helper command, redux, acceptance, tests, Cypress"
 ---
 
-# Acceptance testing
+# Acceptance tests
 
-Volto uses [Cypress](https://www.cypress.io/) for browser-based acceptance testing.
+Volto uses [Cypress](https://www.cypress.io/) for browser-based acceptance tests.
 
-There are a number of fixtures available covering all the configuration use cases.
-These fixtures have both a specific backend and frontend configuration setup and a related set of tests.
-The CI infrastructure runs them all automatically on every push to a branch or PR.
+There are a number of tests available covering all the configuration use cases.
+These tests have both a specific backend and frontend configuration setup and a related set of tests.
+The continuous integration infrastructure runs them all automatically on every push to a branch or a pull request.
 
 The tests can be run in headless mode (same as the CI does), or within the Cypress user interface.
 The latter is the one that you run under development.
 
+
 ## How to run acceptance tests locally (during development)
 
 When writing new acceptance tests, you usually want to minimize the time it takes to run the tests, while also being able to debug or inspect what's going on.
 
 Being able to restart individual components also comes in handy.
 It's recommended to start three individual terminal sessions, one each for running the Plone backend, the Volto frontend, and the acceptance tests.
+All sessions should start from the `packages/volto` directory.
```

Comment:
This is not true. You can run them from the root too, they have similar proxy commands to `packages/volto`

Reply:
Just because you can do something doesn't mean you should do it. We are documenting to beginners how they can run the acceptance testing and if they really care they can see that the commands expand to the volto package. 
Within the volto package you have everything you need from nees folder to lint to run to test and although power users can do whatever they want from volto repo, well that's a power user move :)

Reply:
I actually didn't recommend one place over another. It was my interpretation of the link that @ichim-david had originally used to refer to this location. This brings up something we need to discuss and decide.

There are quite a lot of differences between the two `Makefile`s. It looks like the _primary_ `Makefile` is at the repo root, based in its inclusion of documentation commands and reading files from the repo root. It also appears that the [one in `packages/volto`](https://github.com/plone/volto/commits/main/packages/volto/Makefile) has not been kept up to date with [that at the repo root](https://github.com/plone/volto/commits/main/Makefile), based on the number of updates to each. For example, I would not run `make build` from the package, but I would from the root. Here's another significant difference where some commands fail when spaces are in the `realpath`, which is [something that often bites first-timers on their bum](https://community.plone.org/t/error-running-training-repo-seek-advice-on-pull-request/18618).

```diff
- CHECKOUT_BASENAME=$(shell basename $(shell realpath ./))
+ CHECKOUT_BASENAME="$(shell basename $(shell realpath ./))"
- CHECKOUT_TMP_ABS=$(shell realpath $(CHECKOUT_TMP))
+ CHECKOUT_TMP_ABS="$(shell realpath $(CHECKOUT_TMP))"
```

After looking at these differences as they are today, I think that we should recommend all commands should be run from the root of the repo. We can also suggest a tip that they may be run from the `packages/volto` directory, but there is no guarantee that they will work, such as `make docs-*`. That is, unless and until, we decide that the `Makefile` in the `packages/volto` directory is maintained for full parity with that at the root of the repo.

@sneridagh is that interpretation of `Makefile`s correct?

Reply:
> This is not true. You can run them from the root too, they have similar proxy commands to `packages/volto`

Not quite, https://github.com/plone/volto/blob/main/package.json#L14-L23 although yes in the root you have commands for lint, prettier and stylelint and the :fix options they run for all of the apps and packages defined in the monorepo while the commands from packages/volto only run for the volto core package. 
Yes, you can also manually run the filter commands from the root but then you lose the convenience of simply writing pnpm command and getting the right outcome.

We can enhance the docs with another section for the apps and other packages from the volto umbrella but now we should ensure that we have the docs for how to contribute to Volto core in my opinion.